transit_odp/timetables/extract.py

```diff
@@ -201,9 +205,12 @@ def extract_services(self):
                 message=message,
             )
 
-        df["file_id"] = self.file_id
-        df.set_index(["file_id", "service_code"], inplace=True)
-        return df
+        services_df["file_id"] = self.file_id
+        services_df.set_index(["file_id", "service_code"], inplace=True)
+
+        lines_df["file_id"] = self.file_id
```

Comment:
When feature flag is toggled `lines_df` can be empty. An if condition is required to set the index when the dataframe is not empty